static/js/apps/explore_landing/topics.json

```diff
@@ -26,13 +26,64 @@
             "url": "oq=Health+equity+in+Santa+Clara+County&t=dc/topic/HealthEquity&p=geoId/06085"
           }
         ],
-        "specific": [],
-        "comparison": []
+        "specific": [
+          {
+            "title": "Number of poor hispanic women in Santa Clara County",
+            "url": "oq=Number+of+poor+hispanic+women+in+Santa+Clara+County&p=geoId/06085"
```

Comment:
Can we skip `p=` unless there's also a `t=` ?

Try this URL, and it won't route you to the variable.

Reply:
Done.